cli/commands.go

```diff
@@ -28,6 +28,9 @@ const (
 	defaultServerCount = 1
 )
 
+// Bind mount /dev/mapper for linux users who have lvm/luks/etc in use.
+var defaultBindMounts = []string{"/dev/mapper:/dev/mapper"}
```

Comment:
Should we just add a conditional to check if this path exists? If not (e.g. like you said on windows machines), we simply don't even try to mount it.

Reply:
Oh yeah, good idea!

Reply:
Implemented!